servers/republik/modules/crowdfundings/graphql/schema-types.js

```diff
@@ -165,6 +165,7 @@ type Membership {
   graceEndDate: DateTime
   overdue: Boolean!
   cancellations: [Cancellation!]!
+  canManuallyAddPeriod: Boolean!
```

Comment:
Love that flag.

I urge you to rename as close as possible to its sibling mutation as to have one mental hoop less to jump through.

So, here is a list of these:

- `canAddPeriod` (prop), `addPeriod` (mutation)
- `canAppendPeriod`, `appendPeriod`  ⭐
- `canManuallyAddPeriod`, `addPeriod`
- `canManuallyAppendPeriod`, `appendPeriod`
- `canAddMembershipPeriod`, `addMembershipPeriod`
- `canAppendMembershipPeriod`, `appendMembershipPeriod` ⭐

Those I would pick myself, I marked with a  ⭐.

Here some reasoning around `manual` and `toMembership`, or why I favor dropping it:

- There is no other entity you can add a period to.
- All mutation are somewhat manual.
- Using entity name itself (`MembershipPeriod`) is good. One hoop less.